src/components/FoodOrderComponents/UsersOrderStatus.vue

```diff
@@ -0,0 +1,144 @@
+        v-model="user.isOrderingToday"
+        @change="updateOrdersAmount(user.isOrderingToday)"/>
+    </v-container>
+    <v-divider />
+    <v-footer color="transparent" class="footer">
+          <h5>{{`מספר מזמינים: ${this.ordersAmount}`}}</h5>
+          <v-btn @click="onNextButtonClicked">
+          המשך
+          </v-btn>
+          <h5 class='error-message'>{{this.errorMessage}}</h5>
+    </v-footer>
+  </v-content>
+</template>
+
+<script lang="ts">
+import { isToday } from 'date-fns';
+import { getModule } from 'vuex-module-decorators';
+import {
+  Component, Vue,
+} from 'vue-property-decorator';
+
+import User from '@/models/User';
+import StoreModule from '@/store/storeModule';
+import { AllUsers } from '@/db-service/Users/queries';
+import { updateLastFoodOrder } from '@/db-service/Users/mutations';
+import { dbErrorMessage } from '@/utils/errorMessages';
+
+interface UserOrderStatus {
+  name: string;
+  isLoggedInUser: boolean;
+  isOrderingToday: boolean;
+}
+
+@Component({
+  apollo: {
+    allUsers: {
+      query: AllUsers,
+      fetchPolicy: 'network-only',
+    },
+  },
+})
+export default class UsersOrderStatus extends Vue {
+  private ordersAmount = 0;
+
+  private errorMessage = '';
+
+  private storeModule = getModule(StoreModule, this.$store);
+
+  get users(): UserOrderStatus[] {
+    if (this.$data.allUsers !== undefined) {
+      const users: User[] = this.$data.allUsers.nodes;
+      const userOrderStatus: UserOrderStatus[] = users.map((user: User) => {
+        const isLoggedInUser: boolean = this.isLoggedInUser(user);
+        return {
+          name: user.name,
+          isLoggedInUser,
+          isOrderingToday: UsersOrderStatus.isUserOrderingToday(user, isLoggedInUser),
+        };
+      });
+      this.ordersAmount = userOrderStatus.filter((user) => user.isOrderingToday).length;
+      return userOrderStatus;
+    }
+    return [];
+  }
+
+  get loggedInUser(): User {
+    return this.storeModule.user;
+  }
+
+  private updateOrdersAmount(isLoggedInUserOrdering: boolean) {
+    this.ordersAmount = isLoggedInUserOrdering
+      ? this.ordersAmount + 1
+      : this.ordersAmount - 1;
+  }
+
+  private isLoggedInUser(user: User): boolean {
+    return this.loggedInUser.id === user.id;
+  }
+
+  private onNextButtonClicked(): void {
+    const loggedInUserStatus: UserOrderStatus | undefined = this.users
+      .find((user: UserOrderStatus) => user.isLoggedInUser);
+    if (loggedInUserStatus) {
+      this.$apollo.mutate({
+        mutation: updateLastFoodOrder,
+        variables: {
+          id: this.loggedInUser.id,
+          lastFoodOrder: loggedInUserStatus.isOrderingToday ? new Date() : null,
+        },
+      })
+        .then(() => { this.$emit('passToNextStep'); })
+        .catch(() => { this.errorMessage = dbErrorMessage; });
+    }
+  }
+
+  static isUserOrderingToday(user: User, isLoggedInUser: boolean): boolean {
+    return isLoggedInUser || isToday(new Date(user.lastFoodOrder));
```

Comment:
Why do you need the isLoggedInUser term?

Reply:
Because when initiating the users status the current user’s status is always true, I will change the name to be clearer

Reply:
changed the funstion name at bca91367f175c72c273139b52c0106812fc76efe